### PR TITLE
fix the number of started threads of certain type.

### PR DESCRIPTION
**Pull request by oknet**

### CODE REVIEW

**Comment by randall** [comment]
[approve ci autest]

**Comment by zwoop** [comment]
This seems fine, and I assume is necessary for the changes in #4086 ?

**Comment by SolidWallOfCode** [comment]
Maybe. Looks like an independent bug to me.

File: iocore/eventsystem/UnixEventProcessor.cc [hunk 1]
```diff
@@ -394,8 +394,8 @@ EventProcessor::initThreadState(EThread *t)
 {
   // Run all thread type initialization continuations that match the event types for this thread.
   for (int i = 0; i < MAX_EVENT_TYPES; ++i) {
-    thread_group[i]._started++;
     if (t->is_event_type(i)) { // that event type done here, roll thread start events of that type.
+      thread_group[i]._started++;
```
[the remark this answers is not in the document]
Comment (oknet) [reply iocore/eventsystem/UnixEventProcessor.cc]: @SolidWallOfCode please confirm it.

[the remark this answers is not in the document]
Comment (oknet) [reply iocore/eventsystem/UnixEventProcessor.cc]: updated with `++thread_group[i]._started;`

**Comment by zwoop** [comment]
Cherry-picked to 8.0.x

**Comment by zwoop** [comment]
Is this relevant to 7.x too ?

**Comment by zwoop** [comment]
I'm going to remove the 7.1.5 Project on this, since going forward, we will need to make a second PR against the 7.1.x for proposed back ports.

**Pull request merged**